src/view/com/posts/FeedItem.tsx

```diff
@@ -179,7 +176,27 @@ export const FeedItem = observer(function FeedItemImpl({
         </View>
 
         <View style={{paddingTop: 12}}>
-          {item.reasonRepost && (
+          {source ? (
+            <Link
+              title={sanitizeDisplayName(source.displayName)}
+              href={source.uri}>
+              <Text
+                type="sm-bold"
+                style={pal.textLight}
+                lineHeight={1.2}
+                numberOfLines={1}>
+                From{' '}
+                <DesktopWebTextLink
+                  type="sm-bold"
+                  style={pal.textLight}
+                  lineHeight={1.2}
+                  numberOfLines={1}
+                  text={sanitizeDisplayName(source.displayName)}
+                  href={source.uri}
+                />
+              </Text>
+            </Link>
```

Comment:
What can we do to make this source text feel more like it applies to the following post(s) and not as a part of the top posts's UI? I think a little more whitespace on bottom, maybe align to left of avis? I liked the icon you had in there once.

<img width="718" alt="Screen Shot 2023-09-15 at 2 03 47 PM" src="https://github.com/bluesky-social/social-app/assets/4732330/ac1ef4f0-f8ad-44fa-a2a2-abf4c75e8367">

Reply:
Gonna sit on this one for a bit, haven't found anything I like yet